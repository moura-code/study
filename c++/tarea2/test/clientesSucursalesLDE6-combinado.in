crearClientesSucursalesLDE

crearClientesABB
crearCliente 4 Zafiro Perez 28
agregarAClientesABB
crearCliente 6 Artemio Ruiz 30
agregarAClientesABB
agregarAClientesSucursalesLDE

crearClientesABB
crearCliente 6 Artemio Ruiz 30
agregarAClientesABB
crearCliente 2 Ubaldo Perez 32
agregarAClientesABB
agregarAClientesSucursalesLDE

crearClientesABB
crearCliente 3 Valquiria Fernandez 22
agregarAClientesABB
crearCliente 6 Artemio Ruiz 30
agregarAClientesABB
agregarAClientesSucursalesLDE

crearClientesABB
crearCliente 3 Valquiria Fernandez 22
agregarAClientesABB
crearCliente 5 Zenobia Rodriguez 38
agregarAClientesABB
agregarAClientesSucursalesLDE

cantidadClientesSucursalesLDE
obtenerPrimeroClientesSucursalesLDE
obtenerNesimoClientesSucursalesLDE 3
imprimirClientesSucursalesLDE
imprimirInvertidaClientesSucursalesLDE
removerUltimoClientesSucursalesLDE
clienteMasRepetido

liberarClientesSucursalesLDE


crearClientesSucursalesLDE

crearClientesABB
crearCliente 7 Esmeralda Sanchez 26
agregarAClientesABB
crearCliente 5 Zenobia Rodriguez 38
agregarAClientesABB
agregarAClientesSucursalesLDE

crearClientesABB
crearCliente 5 Zenobia Rodriguez 38
agregarAClientesABB
crearCliente 4 Zafiro Perez 28
agregarAClientesABB
agregarAClientesSucursalesLDE

crearClientesABB
crearCliente 7 Esmeralda Sanchez 26
agregarAClientesABB
crearCliente 3 Valquiria Fernandez 22
agregarAClientesABB
agregarAClientesSucursalesLDE

crearClientesABB
crearCliente 4 Zafiro Perez 28
agregarAClientesABB
crearCliente 1 Rufino Sanchez 24
agregarAClientesABB
agregarAClientesSucursalesLDE

removerNesimoClientesSucursalesLDE 2
imprimirClientesSucursalesLDE
imprimirInvertidaClientesSucursalesLDE
clienteMasRepetido

liberarClientesSucursalesLDE

Fin
