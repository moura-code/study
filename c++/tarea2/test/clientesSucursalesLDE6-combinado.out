Bienvenido al programa principal de Mercado FINGer. Por favor ingrese su comando:
1> La SucursalesLDE de clientes ha sido creado de forma exitosa.
2> El clientesABB ha sido creado de forma exitosa.
3> Cliente creado de forma exitosa.
4> Se ha agregado el cliente al clientesABB de forma exitosa.
5> Cliente creado de forma exitosa.
6> Se ha agregado el cliente al clientesABB de forma exitosa.
7> Se ha agregado el clientesABB a SucursalesLDE de forma exitosa.
8> El clientesABB ha sido creado de forma exitosa.
9> Cliente creado de forma exitosa.
10> Se ha agregado el cliente al clientesABB de forma exitosa.
11> Cliente creado de forma exitosa.
12> Se ha agregado el cliente al clientesABB de forma exitosa.
13> Se ha agregado el clientesABB a SucursalesLDE de forma exitosa.
14> El clientesABB ha sido creado de forma exitosa.
15> Cliente creado de forma exitosa.
16> Se ha agregado el cliente al clientesABB de forma exitosa.
17> Cliente creado de forma exitosa.
18> Se ha agregado el cliente al clientesABB de forma exitosa.
19> Se ha agregado el clientesABB a SucursalesLDE de forma exitosa.
20> El clientesABB ha sido creado de forma exitosa.
21> Cliente creado de forma exitosa.
22> Se ha agregado el cliente al clientesABB de forma exitosa.
23> Cliente creado de forma exitosa.
24> Se ha agregado el cliente al clientesABB de forma exitosa.
25> Se ha agregado el clientesABB a SucursalesLDE de forma exitosa.
26> La cantidad de clientesABB en SucursalesLDE es 4
27> Primer clientesABB:
Cliente Valquiria Fernandez
Id: 3
Edad: 22
Cliente Artemio Ruiz
Id: 6
Edad: 30
28> ClientesABB en la posicion 3:
Cliente Valquiria Fernandez
Id: 3
Edad: 22
Cliente Zenobia Rodriguez
Id: 5
Edad: 38
29> clientesSucursalesLDE de grupos:
Grupo con edad promedio 26.00:
Cliente Valquiria Fernandez
Id: 3
Edad: 22
Cliente Artemio Ruiz
Id: 6
Edad: 30
Grupo con edad promedio 29.00:
Cliente Zafiro Perez
Id: 4
Edad: 28
Cliente Artemio Ruiz
Id: 6
Edad: 30
Grupo con edad promedio 30.00:
Cliente Valquiria Fernandez
Id: 3
Edad: 22
Cliente Zenobia Rodriguez
Id: 5
Edad: 38
Grupo con edad promedio 31.00:
Cliente Ubaldo Perez
Id: 2
Edad: 32
Cliente Artemio Ruiz
Id: 6
Edad: 30
30> clientesSucursalesLDE de grupos:
Grupo con edad promedio 31.00:
Cliente Ubaldo Perez
Id: 2
Edad: 32
Cliente Artemio Ruiz
Id: 6
Edad: 30
Grupo con edad promedio 30.00:
Cliente Valquiria Fernandez
Id: 3
Edad: 22
Cliente Zenobia Rodriguez
Id: 5
Edad: 38
Grupo con edad promedio 29.00:
Cliente Zafiro Perez
Id: 4
Edad: 28
Cliente Artemio Ruiz
Id: 6
Edad: 30
Grupo con edad promedio 26.00:
Cliente Valquiria Fernandez
Id: 3
Edad: 22
Cliente Artemio Ruiz
Id: 6
Edad: 30
31> Ultimo clientesABB (removido):
Cliente Ubaldo Perez
Id: 2
Edad: 32
Cliente Artemio Ruiz
Id: 6
Edad: 30
32> El cliente que aparece en la mayor cantidad de sucursales es:
Cliente Valquiria Fernandez
Id: 3
Edad: 22
33> SucursalesLDE de ClientesABB liberada
34> La SucursalesLDE de clientes ha sido creado de forma exitosa.
35> El clientesABB ha sido creado de forma exitosa.
36> Cliente creado de forma exitosa.
37> Se ha agregado el cliente al clientesABB de forma exitosa.
38> Cliente creado de forma exitosa.
39> Se ha agregado el cliente al clientesABB de forma exitosa.
40> Se ha agregado el clientesABB a SucursalesLDE de forma exitosa.
41> El clientesABB ha sido creado de forma exitosa.
42> Cliente creado de forma exitosa.
43> Se ha agregado el cliente al clientesABB de forma exitosa.
44> Cliente creado de forma exitosa.
45> Se ha agregado el cliente al clientesABB de forma exitosa.
46> Se ha agregado el clientesABB a SucursalesLDE de forma exitosa.
47> El clientesABB ha sido creado de forma exitosa.
48> Cliente creado de forma exitosa.
49> Se ha agregado el cliente al clientesABB de forma exitosa.
50> Cliente creado de forma exitosa.
51> Se ha agregado el cliente al clientesABB de forma exitosa.
52> Se ha agregado el clientesABB a SucursalesLDE de forma exitosa.
53> El clientesABB ha sido creado de forma exitosa.
54> Cliente creado de forma exitosa.
55> Se ha agregado el cliente al clientesABB de forma exitosa.
56> Cliente creado de forma exitosa.
57> Se ha agregado el cliente al clientesABB de forma exitosa.
58> Se ha agregado el clientesABB a SucursalesLDE de forma exitosa.
59> clientesABB en la posicion 2 (removido):
Cliente Rufino Sanchez
Id: 1
Edad: 24
Cliente Zafiro Perez
Id: 4
Edad: 28
60> clientesSucursalesLDE de grupos:
Grupo con edad promedio 24.00:
Cliente Valquiria Fernandez
Id: 3
Edad: 22
Cliente Esmeralda Sanchez
Id: 7
Edad: 26
Grupo con edad promedio 32.00:
Cliente Zenobia Rodriguez
Id: 5
Edad: 38
Cliente Esmeralda Sanchez
Id: 7
Edad: 26
Grupo con edad promedio 33.00:
Cliente Zafiro Perez
Id: 4
Edad: 28
Cliente Zenobia Rodriguez
Id: 5
Edad: 38
61> clientesSucursalesLDE de grupos:
Grupo con edad promedio 33.00:
Cliente Zafiro Perez
Id: 4
Edad: 28
Cliente Zenobia Rodriguez
Id: 5
Edad: 38
Grupo con edad promedio 32.00:
Cliente Zenobia Rodriguez
Id: 5
Edad: 38
Cliente Esmeralda Sanchez
Id: 7
Edad: 26
Grupo con edad promedio 24.00:
Cliente Valquiria Fernandez
Id: 3
Edad: 22
Cliente Esmeralda Sanchez
Id: 7
Edad: 26
62> El cliente que aparece en la mayor cantidad de sucursales es:
Cliente Zenobia Rodriguez
Id: 5
Edad: 38
63> SucursalesLDE de ClientesABB liberada
64> Fin.
